cocotb/share/include/cocotb_utils.h

```diff
@@ -44,9 +44,11 @@ extern void* utils_dyn_open(const char* lib_name);
 extern void* utils_dyn_sym(void *handle, const char* sym_name);
 
 extern int context;
+#define PYTHON_CONTEXT  1
+#define SIM_CONTEXT     0
```

Comment:
An even easier solution would be renaming the `context` variable to `is_python_context`.